src/core/getFrameValidatedMessage.ts

```diff
@@ -18,9 +19,9 @@ export function getHubClient(): HubRpcClient {
  *
  * @param body The JSON received by server on frame callback
  */
-async function getFrameValidatedMessage(body: {
```

Comment:
Open question should we keep this as `getFrameValidatedMessage` or have `getFrameMessage`.
As now we are returing `isValid`.

Reply:
Also should is `data` more correct than `message`?

Reply:
agree and agree .. pr updated